pcawg_metadata_parser/parse_gnos_xml.py

```diff
@@ -1880,18 +1880,18 @@ def bam_aggregation(bam_files):
                     aliquot_tmp = create_aggregated_rna_bam_info(bam)
                     alignment_status['tophat'] = aliquot_tmp
 
-                elif alignment_status.get('tophat').get('gnos_info').get('gnos_id') == bam['bam_gnos_ao_id']:
-                    if bam['gnos_repo'] in alignment_status.get('tophat').get('gnos_info').get('gnos_repo'):
+                elif alignment_status.get('tophat').get('aligned_bam').get('gnos_id') == bam['bam_gnos_ao_id']:
```

Comment:
There are report scripts use the `gnos_info` field, all of these scripts need to be updated accordingly with `aligned_bam`